app/models/miq_schedule_worker/jobs.rb

```diff
@@ -120,7 +120,6 @@ def archived_entities_purge_timer
     queue_work(:class_name => "ContainerGroup", :method_name => "purge_timer", :zone => nil)
     queue_work(:class_name => "ContainerImage", :method_name => "purge_timer", :zone => nil)
     queue_work(:class_name => "ContainerProject", :method_name => "purge_timer", :zone => nil)
-    queue_work(:class_name => "ContainerDefinition", :method_name => "purge_timer", :zone => nil)
```

Comment:
assume theres an issue. 
1) no need to actually purge the defs since we drop the entire table
2) worker tries to purge container_defs, errors out, doesnt try again.

im not sure if 2 is a problem or not